File: NEsper.IO/IllegalStateTransitionException.cs
using System;

using net.esper.client;

namespace net.esper.adapter
{
	/// <summary>
	/// Thrown when an illegal Adapter state transition is attempted.
	/// </summary>
	public class IllegalStateTransitionException : EPException
	{
		/// <summary>
		/// <param name="message">an explanation of the cause of the exception</param>
		/// </summary>
		public IllegalStateTransitionException(String message)
			: base( message )
		{
		}
	}
}

File: trunk/esperio/src/main/java/net/esper/adapter/IllegalStateTransitionException.java
package net.esper.adapter;

import net.esper.client.EPException;

/**
 * Thrown when an illegal Adapter state transition is attempted.
 */
public class IllegalStateTransitionException extends EPException
{
	/**
	 * @param message - an explanation of the cause of the exception
	 */
	public IllegalStateTransitionException(String message)
	{
		super(message);
	}
}
